develop/tutorials/articles/280-workflow/01-creating-workflow-definitions/03-notifications.markdown

```diff
@@ -48,38 +55,56 @@ processing is leaving a node (`onExit`). If you specify a notification to
 be sent on assignment, the assignee is automatically notified.
 
 **Recipients**
-: Decide who should receive the notification in the `<recipients>` tag.
+: Decide who should receive the notification in the `<recipients>` tag:
+
+        <recipients>
+            [SEE BELOW FOR THE AVAILABLE RECIPIENT TAGS]
+        </recipients>
+
 Available recipient tags are 
 
-- `<user />`: notify the User that sent the asset through the workflow using.
+- `<user>`: notify the User that sent the asset through the workflow. Simply
+    specify the tag as `<user />`. To notify a specific user, enter the
+    `userId`:
+
+        <recipients>
+            <user />
+            <user>
+                <user-id>20139</user-id>
+            </user
+        </recipients>
 
 - `<roles>`: notify specific roles, either by ID or by their type and name.
 
-        <roles>
-            <role>
-                <role-id>33621</role-id>
-            </role>
-            <role>
-                <role-type>regular</role-type>
-                <name>Power User</name>
-                <auto-create>false</auto-create>
-            </role>
-        </roles>
+        <recipients>
```

Comment:
Hey @rbohl, according to the 7.1 DTD, the user can't define the recipients type to be _Role_ and _Role type_ simultaneously, only one is supported by time.